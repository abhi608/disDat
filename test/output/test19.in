Starting T1
Starting T2
Starting T3
Starting T4
Starting T5
T3 got read lock on x3 having value 30
Site: 4 failed
T3 aborted as site 4 failed
Site: 4 recovered
DEBUG| Clearing site locks for T3 variable: x3
Clearing locks for T3 variable: x3
T4 got read lock on x4 having value 40
T5 got read lock on x5 having value 50
T1 got read lock on x1 having value 10
T2 got read lock on x2 having value 20
DEBUG| T1 did not get write lock on x2 site: 1
T1 is blocked for a write lock by T2 on x2
T2 got write lock on x3
DEBUG| T5 did not get write lock on x1 site: 2
T5 is blocked for a write lock by T1 on x1
DEBUG| T5 did not get write lock on x1 site: 2
T5 is blocked for a write lock by T1 on x1
T5 read the value 50 of variable x5
T5 committed
DEBUG| Clearing site locks for T5 variable: x5
Clearing locks for T5 variable: x5
T4 got write lock on x5
T4 read the value 40 of variable x4
T4 committed
DEBUG| Clearing site locks for T4 variable: x4
Clearing locks for T4 variable: x4
DEBUG| Clearing site locks for T4 variable: x5
Clearing locks for T4 variable: x5
T2 read the value 20 of variable x2
T2 committed
DEBUG| Clearing site locks for T2 variable: x2
Clearing locks for T2 variable: x2
DEBUG| Clearing site locks for T2 variable: x3
Clearing locks for T2 variable: x3
T1 got write lock on x2
T1 read the value 10 of variable x1
T1 committed
DEBUG| Clearing site locks for T1 variable: x1
Clearing locks for T1 variable: x1
DEBUG| Clearing site locks for T1 variable: x2
Clearing locks for T1 variable: x2
=== Site 1 ===
x2: 10 at site 1
All other variables have same initial value
=== Site 2 ===
x2: 10 at site 2
All other variables have same initial value
=== Site 3 ===
x2: 10 at site 3
All other variables have same initial value
=== Site 4 ===
x20 is not available for reading
x3: 20 (available at site 4 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x14 is not available for reading
x12 is not available for reading
x16 is not available for reading
x13: 130 (available at site 4 for reading as it is the only copy or has been written after recovery)
x10 is not available for reading
x4 is not available for reading
x2: 10 (available at site 4 for reading as it is the only copy or has been written after recovery)
x8 is not available for reading
x6 is not available for reading
=== Site 5 ===
x2: 10 at site 5
All other variables have same initial value
=== Site 6 ===
x5: 40 at site 6
x2: 10 at site 6
All other variables have same initial value
=== Site 7 ===
x2: 10 at site 7
All other variables have same initial value
=== Site 8 ===
x2: 10 at site 8
All other variables have same initial value
=== Site 9 ===
x2: 10 at site 9
All other variables have same initial value
=== Site 10 ===
x2: 10 at site 10
All other variables have same initial value
