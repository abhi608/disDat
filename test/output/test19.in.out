Starting T1
Starting T2
Starting T3
Starting T4
Starting T5
T3 got read lock on x3 having value 30
Site: 4 failed
T3 aborted as site 4 failed
Site: 4 recovered
DEBUG| Clearing site locks for T3 variable: x3
Clearing locks for T3 variable: x3
T4 got read lock on x4 having value 40
T5 got read lock on x5 having value 50
T1 got read lock on x1 having value 10
T2 got read lock on x2 having value 20
DEBUG| T1 did not get write lock on x2 site: 1
T1 is blocked for a write lock by T2 on x2
T2 got write lock on x3
DEBUG| T5 did not get write lock on x1 site: 2
T5 is blocked for a write lock by T1 on x1
DEBUG| T5 did not get write lock on x1 site: 2
T5 is blocked for a write lock by T1 on x1
T5 read the value 50 of variable x5
T5 committed
DEBUG| Clearing site locks for T5 variable: x5
Clearing locks for T5 variable: x5
T4 got write lock on x5
T4 read the value 40 of variable x4
T4 write the value 40 of variable x5
T4 committed
DEBUG| Clearing site locks for T4 variable: x4
Clearing locks for T4 variable: x4
DEBUG| Clearing site locks for T4 variable: x5
Clearing locks for T4 variable: x5
T2 read the value 20 of variable x2
T2 write the value 20 of variable x3
T2 committed
DEBUG| Clearing site locks for T2 variable: x2
Clearing locks for T2 variable: x2
DEBUG| Clearing site locks for T2 variable: x3
Clearing locks for T2 variable: x3
T1 got write lock on x2
T1 read the value 10 of variable x1
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 committed
DEBUG| Clearing site locks for T1 variable: x1
Clearing locks for T1 variable: x1
DEBUG| Clearing site locks for T1 variable: x2
Clearing locks for T1 variable: x2
=== Site 1 ===
x20: 200 at site 1
x16: 160 at site 1
x14: 140 at site 1
x10: 100 at site 1
x8: 80 at site 1
x6: 60 at site 1
x4: 40 at site 1
x18: 180 at site 1
x12: 120 at site 1
x2: 10 at site 1
=== Site 2 ===
x20: 200 at site 2
x16: 160 at site 2
x14: 140 at site 2
x8: 80 at site 2
x6: 60 at site 2
x11: 110 at site 2
x4: 40 at site 2
x18: 180 at site 2
x12: 120 at site 2
x2: 10 at site 2
x10: 100 at site 2
x1: 10 at site 2
=== Site 3 ===
x20: 200 at site 3
x16: 160 at site 3
x14: 140 at site 3
x10: 100 at site 3
x8: 80 at site 3
x6: 60 at site 3
x4: 40 at site 3
x18: 180 at site 3
x12: 120 at site 3
x2: 10 at site 3
=== Site 4 ===
x20 is not available for reading
x16 is not available for reading
x14 is not available for reading
x10 is not available for reading
x6 is not available for reading
x4 is not available for reading
x13: 130 (available at site 4 for reading as it is the only copy or has been written after recovery)
x8 is not available for reading
x3: 20 (available at site 4 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x12 is not available for reading
x2: 10 (available at site 4 for reading as it is the only copy or has been written after recovery)
=== Site 5 ===
x20: 200 at site 5
x16: 160 at site 5
x14: 140 at site 5
x10: 100 at site 5
x8: 80 at site 5
x6: 60 at site 5
x4: 40 at site 5
x18: 180 at site 5
x12: 120 at site 5
x2: 10 at site 5
=== Site 6 ===
x20: 200 at site 6
x14: 140 at site 6
x15: 150 at site 6
x10: 100 at site 6
x8: 80 at site 6
x6: 60 at site 6
x16: 160 at site 6
x5: 40 at site 6
x4: 40 at site 6
x18: 180 at site 6
x12: 120 at site 6
x2: 10 at site 6
=== Site 7 ===
x20: 200 at site 7
x16: 160 at site 7
x14: 140 at site 7
x10: 100 at site 7
x8: 80 at site 7
x6: 60 at site 7
x4: 40 at site 7
x18: 180 at site 7
x12: 120 at site 7
x2: 10 at site 7
=== Site 8 ===
x17: 170 at site 8
x16: 160 at site 8
x14: 140 at site 8
x10: 100 at site 8
x8: 80 at site 8
x20: 200 at site 8
x7: 70 at site 8
x6: 60 at site 8
x4: 40 at site 8
x18: 180 at site 8
x12: 120 at site 8
x2: 10 at site 8
=== Site 9 ===
x20: 200 at site 9
x16: 160 at site 9
x14: 140 at site 9
x10: 100 at site 9
x8: 80 at site 9
x6: 60 at site 9
x4: 40 at site 9
x18: 180 at site 9
x12: 120 at site 9
x2: 10 at site 9
=== Site 10 ===
x16: 160 at site 10
x19: 190 at site 10
x14: 140 at site 10
x10: 100 at site 10
x20: 200 at site 10
x9: 90 at site 10
x8: 80 at site 10
x6: 60 at site 10
x4: 40 at site 10
x18: 180 at site 10
x12: 120 at site 10
x2: 10 at site 10
